tests/plugins/indentlist/manual/taberrorininlineeditor.html

```diff
@@ -0,0 +1,40 @@
+<head>
+	<script>
+		if ( !CKEDITOR.env.chrome ) {
```

Comment:
Interesting.
when I fired the test from link:
http://tests.ckeditor.dev:1030/#tests/path:/tests/plugins/indentlist/manual with a play button it seemd to be ignored, but there is no notification when test is opened directly.

I moved condition to body to work properly.